Source/Scene/Scene.js

```diff
@@ -2696,6 +2706,68 @@ define([
         return object;
     };
 
+    var scratchPickDepthPassState;
+
+    function renderForPickDepth(scene, drawingBufferPosition) {
+        var context = scene._context;
+        var us = context.uniformState;
+        var frameState = scene._frameState;
+
+        // Update with previous frame's number and time, assuming that render is called before picking.
+        updateFrameState(scene, frameState.frameNumber, frameState.time);
+        frameState.cullingVolume = getPickCullingVolume(scene, drawingBufferPosition, 1, 1);
+        frameState.passes.render = true;
```

Comment:
> the depth copies for each frustum only happen during the render pass

I'm not following.  What depth copies?

> Another option is to add a new depth pass

Could work, but what about this: could we just derive commands from the previous frame's commands?  Instead of calling all the `update` functions, we rely on the state of the frustums/commands from the previous frame.  We would need to guarantee that WebGL resources pointed to by the commands were not deleted, we could even rename the function to indicate this.